app/views/admin/admin_users/_user_history.html.erb

```diff
@@ -24,6 +24,28 @@
       <td><%= ts("Account Created") %></td>
       <td><%= ts("System Generated") %></td>
     </tr>
+    <tr>
+      <td>
+        <% if @user.current_sign_in_at.nil? %>
+          <%= ts("No login recorded") %>
+        <% else %>
+          <%= @user.current_sign_in_at %> (<%= @user.current_sign_in_ip %>)
+        <% end %>
+      </td>
+      <td><%= ts('Current Login') %></td>
```

Comment:
You should use [double quotes](https://github.com/otwcode/otwarchive/wiki/Coding-standards#general-code-formatting) for all the new additions.